packages/ra-ui-materialui/src/button/Button.tsx

```diff
@@ -42,9 +48,11 @@ export const Button = (props: ButtonProps) => {
         label,
         color = 'primary',
         size = 'small',
+        to: locationDescriptor,
```

Comment:
I don't understand this change. Why do you need to decomponse the to prop now?

Reply:
I wanted to reduce the breaking changes for our users. The `to` prop of react-router Link has changed. It does not include `state` anymore, nor the options such as `replace`